numpy/core/numeric.py

```diff
@@ -2448,6 +2448,10 @@ def allclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
     The above equation is not symmetric in `a` and `b`, so that
     `allclose(a, b)` might be different from `allclose(b, a)` in
     some rare cases.
+        
+    The comparison of `a` and `b` includes standard broadcasting,
+    which means that `a` and `b` need not have the same shape in order for
+    allclose(a, b) to evaluate to True.  (The same is true for `equals`.)
```

Comment:
Double backticks around `allclose(a, b)`. No parenthesis around the closing sentence.